cnab240/bancos/bancodobrasil/specs/spec_to_doc.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import sys
+import json
+from collections import OrderedDict
+
+fname = sys.argv[1]
+f = file(fname)
+spec = json.load(f, object_pairs_hook=OrderedDict)
+f.close()
+
+campos = spec.get('campos')
+for campo in campos.values():
```

Comment:
`campos.values()` vai falhar com attribute error se `spec.get('campos')` for None.

Se `spec.get('campos')` não deve retornar `None`, então o certo é usar:

 - `spec['campos']`

Se `campos` pode estar ausente de `spec`, então o certo é usar:

 - `spec.get('campos', {})`